pkgs/development/libraries/libpst/default.nix

```diff
@@ -9,14 +19,24 @@ stdenv.mkDerivation rec {
     sha256 = "11wrf47i3brlxg25wsfz17373q7m5fpjxn2lr41dj252ignqzaac";
   };
 
-  nativeBuildInputs = [ autoreconfHook pkg-config ];
+  nativeBuildInputs = [
+    autoreconfHook
+    pkg-config
+  ];
+
   buildInputs = [
-    boost libgsf bzip2
-    xmlto gettext imagemagick doxygen
+    bzip2
+    doxygen
```

Comment:
`doxygen` and `xmlto` are likely used for building docs so they should go to `nativeBuildInputs`. And I would expect the same for gettext. And imagick, but that one is actually used at runtime: http://hg.five-ten-sg.com/libpst/file/tip/src/pst2dii.cpp.in#l36

Reply:
For imagemagick do I put it in propagatedBuildInputs, or keep it in buildInputs? It looks like it sets the full path at build time, so I don't think I should remove it entirely right? I always get confused by dependencies, thanks for the help.

Reply:
I think `imagemagick` can stay in `buildInputs` since it will be hardcoded.